docs/ops/sort/TopK_11.md

```diff
@@ -0,0 +1,118 @@
+  * **Default value**: "i32"
+  * **Required**: *no*
+
+
+**Inputs**:
+
+*   **1**: tensor with arbitrary rank and type *T*. **Required.**
+
+*   **2**: The value of *K* -- a scalar of any integer type that specifies how many elements from the input tensor should be selected. **Required.**
+
+**Outputs**:
+
+*   **1**: Output tensor of type *T* with at most *k* values from the input tensor along specified dimension *axis*. The shape of the tensor is `[input1.shape[0], ..., input1.shape[axis-1], 1..k, input1.shape[axis+1], ..., input1.shape[input1.rank - 1]]`.
+
+*   **2**: Output tensor containing indices of the corresponding elements(values) from the first output tensor. The indices point to the location of selected values in the original input tensor. The shape of this output tensor is the same as the shape of the 1st output, that is `[input1.shape[0], ..., input1.shape[axis-1], 1..k, input1.shape[axis+1], ..., input1.shape[input1.rank - 1]]`. The type of this tensor *T_IND* is controlled by the `index_element_type` attribute.
+
+**Types**
+
+* *T*: any numeric type.
+
+* *T_IND*: `int64` or `int32`.
+
+**Detailed Description**
+
+The output tensor is populated by values computed in the following way:
+
+    output[i1, ..., i(axis-1), j, i(axis+1) ..., iN] = top_k(input[i1, ...., i(axis-1), :, i(axis+1), ..., iN]), k, sort, mode)
+
+So for each slice `input[i1, ...., i(axis-1), :, i(axis+1), ..., iN]` *TopK* values are computed individually.
+
+Sorting and minimum/maximum are controlled by `sort` and `mode` attributes with additional configurability provided by `stable`:
+  * *sort*=`value`, *mode*=`max`, *stable*=`false` - descending by value, relative order of equal elements not guaranteed to be maintained
+  * *sort*=`value`, *mode*=`max`, *stable*=`true`  - descending by value, relative order of equal elements guaranteed to be maintained
+  * *sort*=`value`, *mode*=`min`, *stable*=`false` - ascending by value, relative order of equal elements not guaranteed to be maintained
+  * *sort*=`value`, *mode*=`min`, *stable*=`true`  - ascending by value, relative order of equal elements guaranteed to be maintained
+  * *sort*=`index`, *mode*=`max` - descending by index
+  * *sort*=`index`, *mode*=`min` - ascending by index
+  * *sort*=`none` , *mode*=`max` - undefined
+  * *sort*=`none` , *mode*=`min` - undefined
+
+The relative order of equivalent elements in a given slice is only preserved if the *stable* attribute is set to `true`. This makes the implementation use stable sorting algorithm during the computation of TopK elements. Otherwise the output order is undefined.
+
+**Example**
+
+This example assumes that `K` is equal to 10:
+
+```xml
+<layer ... type="TopK" ... >
+    <data axis="3" mode="max" sort="value" stable="true" index_element_type="i64"/>
+    <input>
+        <port id="0">
+            <dim>1</dim>
+            <dim>3</dim>
+            <dim>224</dim>
+            <dim>224</dim>
+        </port>
+        <port id="1">
```

Comment:
Please add the actual value like [here.](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sort/TopK_1.md)

Reply:
There's a comment just above the example XML that `K` is assumed to be equal to 10. This section is supposed to describe the shapes(not tensor values) and I thought that this would be more consistent.